### DISCUSSION TITLE
Reorganising file structure

### DISCUSSION THREAD

**Discussion by faraplay** [OctoNezd/oldlander#47] [Ideas]
Having made some additions and changes to the CSS, I always find it annoying to find the appropriate CSS file to put any changes in, and it's not very clear to me how they're currently organised. Also I imagine that this problem will only get worse, the more we develop.
Would it be better to reorganise the file structure to group the styles with the JS for the components they modify?
E.g. something like this:
```
src/
- posts/
-- css/
--- postIcons.css
-- index.js
-- expando.js
-- expandoButton.js
-- etc.
- header/
-- css/
--- header.css
-- header.js
- sidebar/
-- css/
--- subredditSidebar.css
--- userSidebar.css
-- index.js
-- buildSidebar.js
-- etc.
```

**Comment by OctoNezd** [comment]
Indeed, reorganisation of current structure would be a good idea. I will get on with it.

**Comment by OctoNezd** [comment]
Check latest commit - what do you think?

**Reply by faraplay** [reply]
Yeah it looks a lot better!

**Comment by OctoNezd** [comment]
Pushed now
